### PR TITLE
Convert to WebIDL contiguous mode

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

**Comment by martinthomson** [comment]
There seems to be a bunch of unrelated changes here.

**Comment by dontcallmedom** [comment]
it's all done by my conversion script; are there changes that seem wrong?

File: index.html [hunk 1]
```diff
@@ -77,31 +77,26 @@
...
+      <code><dfn><a href="https://w3c.github.io/mediacapture-main/#idl-def-MediaStreamTrack">MediaStreamTrack</a></dfn></code>
+      is used to identify the media in a <code><dfn><a href="https://w3c.github.io/mediacapture-main/#idl-def-MediaStream">MediaStream</a></dfn></code>.
     </p>
 
-    <dl class="idl" title="partial interface HTMLMediaElement">
-      <dt>MediaStream captureStream()</dt>
-      <dd>
+    <div><pre class="idl">partial interface HTMLMediaElement {
+    MediaStream captureStream ();
+    MediaStream captureStreamUntilEnded ();
```
Comment (yell0wd0g) [inline index.html]: Ah, we decided to remove this fellow in #45.

File: index.html [hunk 2]
```diff
@@ -225,15 +220,11 @@
           unplayable playback rate causes the captured audio track to become
           muted.
         </p>
-      </dd>
-
-      <dt>MediaStream captureStreamUntilEnded()</dt>
-      <dd>
+      <div><em>No parameters.</em></div><div><em>Return type: </em><code>MediaStream</code></div></dd><dt><code>captureStreamUntilEnded</code></dt><dd>
```
Comment (yell0wd0g) [inline index.html]: Same as before: this `div` down to l.245 can safely be removed.

File: index.html [hunk 3]
```diff
@@ -265,9 +255,9 @@
       allow for controlling when frames are sampled from the canvas.
     </p>
 
-    <dl class="idl" title="partial interface HTMLCanvasElement">
-      <dt>MediaStream captureStream(optional double frameRate)</dt>
-      <dd>
+    <div><pre class="idl">partial interface HTMLCanvasElement {
+    MediaStream captureStream (optional double frameRate);
+};</pre><section><h2>Methods</h2><dl data-link-for="HTMLCanvasElement" data-dfn-for="HTMLCanvasElement" class="methods"><dt><dfn><code>captureStream</code></dfn></dt><dd>
```
Comment (yell0wd0g) [inline index.html:238]: nit: Maybe add a line-break between `</pre>` and `<section>` ?

**Review by yell0wd0g — APPROVED** [review]
The changes LGTM with the comments about the disappeared method `captureStreamUntilEnded`.

**Comment by dontcallmedom** [comment]
PR rebased on latest gh-pages

**Comment by yell0wd0g** [comment]
@dontcallmedom did you remove the references to `captureStreamUntilEnded()`? I'm still seeing them in the diffs.

**Comment by dontcallmedom** [comment]
@miguelao ah indeed, I had missed one in my rebase - I think it should be ok now

**Pull request merged**